diff --git a/dist/index.js b/dist/index.js
index 5eefb6d6dffdfd134a267c418923d99674f6cb55..7769faafb55441c321d941f364d1680527cf516d 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1432,7 +1432,7 @@ async function getOptimizeDepsIncludes(rootDir) {
 		const serviceModule = pkg.servicesModuleId;
 		return serviceModule ? [serviceModule] : [];
 	});
-	trailsModules.push(`${RuntimeSupport.RUNTIME_PACKAGE_NAME}/**`);
+	//trailsModules.push(`${RuntimeSupport.RUNTIME_PACKAGE_NAME}/**`);
 	isDebug && debug("Optimizing additional modules %O", trailsModules);
 	return trailsModules;
 }
