diff --git a/dist/index.js b/dist/index.js
index 104ba85620defccd7117ae535e74d65bed8f7b86..839835976ac5ba5ec8d3069971fb796fbd38439e 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1700,7 +1700,7 @@ async function getOptimizeDepsIncludes(rootDir) {
     const serviceModule = pkg2.servicesModuleId;
     return serviceModule ? [serviceModule] : [];
   });
-  trailsModules.push(`${RuntimeSupport3.RUNTIME_PACKAGE_NAME}/**`);
+  //trailsModules.push(`${RuntimeSupport3.RUNTIME_PACKAGE_NAME}/**`);
   isDebug5 && debug6("Optimizing additional modules %O", trailsModules);
   return trailsModules;
 }
