diff --git a/dist/index.js b/dist/index.js
index 6ca17b6a74247e867fe7771eeba78be133c4d9f0..99ddcfb31146d5b93a3d688ba3ac9c4d4e41dbed 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1535,7 +1535,7 @@ async function optimizeTrailsPackages(userConfig, rootDir) {
     const serviceModule = pkg2.servicesModuleId;
     return serviceModule ? [serviceModule] : [];
   });
-  trailsModules.push(`${RuntimeSupport.RUNTIME_PACKAGE_NAME}/**`);
+  //trailsModules.push(`${RuntimeSupport.RUNTIME_PACKAGE_NAME}/**`);
   isDebug4 && debug5("Optimizing additional modules %O", trailsModules);
   const optimizeDeps = userConfig.optimizeDeps ??= {};
   const includes = optimizeDeps.include ??= [];
